inotify/adapters.py

```diff
@@ -166,7 +166,7 @@ def _handle_inotify_event(self, wd):
 
             header = _INOTIFY_EVENT(*header_raw)
             type_names = self._get_event_names(header.mask)
-            _LOGGER.debug("Events received in stream: {}".format(type_names))
+            _LOGGER.debug("Events received in stream: {0}".format(type_names))
```

Comment:
Unnecessary and irrelevant.

Reply:
Wrong, Dustin, you need to explicitly number fields in Python 2.6. From "What's New in Python 2.7":

> The str.format() method now supports automatic numbering of the replacement fields.